docs/UserGuide.md

```diff
@@ -20,8 +20,9 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
 3. Press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
 
 - List
-  - `list /module`: Lists the names of all the recorded modules
-  - `list /module /{module_code}`: Lists all the videos recorded in that module code
+  - `list`: Lists the names of all the recorded modules
+  - `list /mod /{module_code}`: Lists all the lectures in the specified module
+  - `list /mod /{module_code} /lec {lecture_name}`: Lists all the videos in the specified module and lecture
```

Comment:
Is the / before {module_code} supposed to be there?

Reply:
Nope thanks for pointing out!